Doc/includes/email-alternative.py

```diff
@@ -8,14 +8,14 @@
 
 # Create the base text message.
 msg = EmailMessage()
-msg['Subject'] = "Ayons asperges pour le déjeuner"
+msg['Subject'] = "Pourquoi pas des asperges pour ce midi?"
```

Comment:
```suggestion
msg['Subject'] = "Pourquoi pas des asperges pour ce mid i?"
```

French typography have spaces before "?", "!", ":", ...